Core/templates/blog.html

```diff
@@ -191,31 +188,31 @@ <h5 class="font-weight-bold text-uppercase mt-3 mb-4">Clients</h5>
                     
                     <ul class="list-unstyled">
                         <li>
-                            <a href="http://akgec.in/">AKGEC</a>
+                            <a target="_blank" href="https://akgec.ac.in/">AKGEC</a>
```

Comment:
Try to keep unrelated changes in separate pull requests. For eg. this PR is related to registration email feature.